src/setting.rs

```diff
@@ -1128,3 +1161,86 @@ impl PadButtonBrightness {
         Ok((input, value))
     }
 }
+
+/// Found at "MIXER > DJ SETTING > CH FADER CURVE" of the "My Settings" page in the Rekordbox
+/// preferences.
+#[derive(Debug)]
+pub enum ChannelFaderCurve {
+    /// Steep volume raise when the fader is moved near the top.
+    SteepTop,
+    /// Linear volume raise when the fader is moved.
+    Linear,
+    /// Steep volume raise when the fader is moved near the bottom.
+    SteepBottom,
+    /// Unknown value.
+    Unknown(u8),
+}
+
+impl ChannelFaderCurve {
+    fn parse(input: &[u8]) -> IResult<&[u8], Self> {
+        let (input, value) = nom::number::complete::u8(input)?;
+        let value = match value {
+            0x80 => Self::SteepTop,
+            0x81 => Self::Linear,
+            0x82 => Self::SteepBottom,
+            _ => Self::Unknown(value),
```

Comment:
In the long run, it would be nice if we could have some diagnostics for the `Unknown` case. This would help catch edge cases of values we don't know.

Reply:
Yeah, I'm not sure about it either. On the positive side, it allows deserializing and even re-serializing without information loss. On the other hand, it would probably be good to notice such issues early. Unsure what the best way to handle this is.

Reply:
I think we need a way to handle partial failure like this. Lots of the other unknown fields aren't arbitrary, but for example just seem to contain a static value most of the time. It would be nice to have a way of (de-)serializing them while still conveying that the contained value was differing from the expected one. This could help tremendously in helping to find databases where the usual assumptions don't apply.

Reply:
Well, we could add a new method (e.g. `checkForUnexpectedData() -> Result<(), Error>`) that looks for unexpected data in fields. But that is something for another PR.